src/App.tsx

```diff
@@ -1,14 +1,20 @@
-import { RecoilRoot } from "recoil";
 import { Routes, Route } from "react-router-dom";
-import Authentification from "./authentification/Authentification";
-import AuthRegisterForm from "./authentification/components/AuthRegisterForm";
+import Navbar from "./global/components/Navbar";
+import Authentication from "./authentification/Authentication";
+import { useRecoilState } from "recoil";
+import { userState } from "./authentification/state/userState";
 
 export default function App() {
+  const [isLoggedIn, setIsLoggedIn] = useRecoilState(userState);
```

Comment:
You dont need setter function here, just the value.

Reply:
Removed setter function.